ApiDoctor.Validation/MethodDefinition.cs

```diff
@@ -468,24 +481,17 @@ internal static void RewriteHeadersWithParameters(HttpRequest request, IEnumerab
         /// </summary>
         internal void VerifyRequestFormat(IssueLogger issues)
         {
-            HttpParser parser = new HttpParser();
             HttpRequest request;
-            try
+            HttpParser.TryParseHttpRequest(this.Request, out request);
```

Comment:
Looks like you want to pass in the issues here as well when you call `TryParseHttpRequest`

Reply:
At this point, validation of the HttpRequest had already happened and any issues logged. Adding issues here causes duplicated errors if any.